app/views/static_pages/contact.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= provide(:title, 'Contact')%>
+<div class="content jumbotron">
+<h1 class="text-center">Contact us</h1>
```

Comment:
Please correct your indention for clean code.
This is an example of good practice
![image](https://user-images.githubusercontent.com/35232348/103610426-1ea9e100-4f5b-11eb-94da-6b1b999f6b75.png)